study-builder/tenants/rgp/pages/login.html

```diff
@@ -17,63 +18,337 @@
 <script src="https://cdn.auth0.com/js/es5-shim.min.js"></script>
 <![endif]-->
 
-<script src="https://cdn.auth0.com/js/lock/11.27/lock.min.js"></script>
+<script src="https://cdn.auth0.com/js/lock/11.5.2/lock.min.js"></script>
+
+<style type="text/css">
+    body {
+        font-family: 'Roboto', sans-serif;
+    }
+
+    .auth0-lock.auth0-lock .auth0-lock-overlay {
+        background: radial-gradient(#ffffff, #f3f3f2, #f3f3f2);
+        background-image: radial-gradient(#ffffff, #f3f3f2, #f3f3f2);
+        background-color: #fff;
+    }
+
+    .auth0-lock.auth0-lock.auth0-lock-opened .auth0-lock-widget {
+        box-shadow: 0 0 40px 4px #888888;
+    }
+
+    .auth0-lock-content {
+        padding-top:10px !important;
+    }
+
+    .auth0-lock.auth0-lock .auth0-lock-form p {
+        text-align:left;
+    }
+
+    a:link {
+        text-decoration: underline !important;
+    }
+</style>
+
 <script>
     // Decode utf8 characters properly
     var config = JSON.parse(decodeURIComponent(escape(window.atob('@@config@@'))));
     config.extraParams = config.extraParams || {};
-    var connection = config.connection;
-    var prompt = config.prompt;
-    var languageDictionary;
-    var language;
-
-    if (config.dict && config.dict.signin && config.dict.signin.title) {
-        languageDictionary = { title: config.dict.signin.title };
-    } else if (typeof config.dict === 'string') {
-        language = config.dict;
+
+    var inviteKey = 'invitationId';
+
+    // save the invitation in local storage to minimize breached password loss-of-invitation
+    if (config.extraParams.invitation_id) {
+        localStorage.setItem(inviteKey,config.extraParams.invitation_id);
+        console.log('Saving invitation ' + config.extraParams.invitation_id);
+    } else {
+        var invitation = localStorage.getItem(inviteKey);
+        if (invitation) {
+            // unsafe, but better than losing this value during breached password shield
+            config.internalOptions.invitation_id = invitation;
+            console.log('Using stored invitation ' + invitation + ' from local storage');
+        }
     }
+
+    var mode = config.extraParams.mode;
+    var languageCode = config.extraParams.language;
+    var study_guid = config.extraParams.study_guid;
+    var studyColor = '#e8702a';
+    var studyHelpEmail = 'info@prionregistry.org';
```

Comment:
You might want to take advantage of Auth0 CLI rendering and use `##STUDY_EMAIL##` here.

Reply:
This is another thing I don't see being used